pyasic/miners/backends/braiins_os.py

```diff
@@ -40,6 +40,8 @@
 from pyasic.web.braiins_os import BOSerWebAPI, BOSMinerWebAPI
 from pyasic.web.braiins_os.proto.braiins.bos.v1 import SaveAction
 from pyasic.data.pools import PoolMetrics
+from pyasic.web.braiins_os.boser import _get_pools_Group
+from grpclib.client import Channel
```

Comment:
Dont need these imports.  Just use the miner `web` attribute and make calls directly on that.